vision/common/constants.py

```diff
@@ -23,6 +23,25 @@
 Corners: TypeAlias = NDArray[Shape["4, 2"], Float64]
 
 
+class Location(TypedDict):
+    """
+    A saved coordinate location - primarily used for ODLCs
+
+    Attributes
+    ----------
+    latitude: float
+        The latitude in degrees of the location
+    longitude: float
+        The longitude in degrees of the location
+    """
+
+    latitude: float
+    longitude: float
```

Comment:
Should update `StandardObject` in vision > common > odlc_characteristics.py to use this. Also, that file might be a better place to put this.

Reply:
The `StandardObject` class is not used anywhere, and we are already using `BoundingBox` for the same purpose. Also, odlc_characteristics.py stores information about  ODLC objects, not generic typedefs and declarations. (Bring questions to @EnderDude67 for clarification.)